ibm_catalog.json

```diff
@@ -265,7 +264,7 @@
           "dependencies": [
             {
               "name": "deploy-arch-ibm-account-infra-base",
-              "description": "This module prepares your IBM Cloud account with the necessary configurations to ensure a secure and organized environment for your architecture.",
+              "description": "Cloud automation for Account Configuration organises your IBM Cloud account with a ready-made set of resource groups by default—and, when you enable the \"with Account Settings\" option, it also applies baseline security and governance settings.",
```

Comment:
Did we run this by the content team? "organises" is the European spelling - we might need to use US?

Reply:
We can change the spelling. After migration, the content team is going to review, and we can update as needed.